ci/requirements/py36-min-all-deps.yml

```
@@ -13,7 +13,7 @@ dependencies:
   - cartopy=0.17
   - cdms2=3.1
   - cfgrib=0.9
-  - cftime=1.0.3  # FIXME need 1.0.5 (not released yet); 1.0.4 is broken
+  - cftime
```

I think we may want to keep the version in this particular file pinned at `1.0.3` as it represents the new minimum version of cftime we will test against for the time being; xref: https://github.com/pydata/xarray/pull/3431#discussion_r337606766.
```suggestion
  - cftime=1.0.3
```

---

As per our versions policy, we only support the latest available patch version
http://xarray.pydata.org/en/stable/installing.html#minimum-dependency-versions